akka-cluster-sharding/src/main/scala/akka/cluster/sharding/protobuf/ClusterShardingMessageSerializer.scala

```diff
@@ -68,6 +68,7 @@ private[akka] class ClusterShardingMessageSerializer(val system: ExtendedActorSy
   private val ShardStoppedManifest = "BK"
   private val GracefulShutdownReqManifest = "BL"
   private val RegionStoppedManifest = "BM"
+  private val ShardHomesManifest = "BN"
```

Comment:
it should be ok for rolling updates to add this, it will be discarded in old version when it's not known

Reply:
In the past, on a similar situation, we added a note on the release notes. 
https://akka.io/blog/news/2021/08/19/akka-2.6.16-released

Reply:
Good point, let's remember to add that in the 2.6.18 release notes.

Reply:
So, back then, I created a draft PR for the release notes to not forget it, but that was during the vacation period and we had some communication breakdown and we missed the draft PR. 

I think it's good to create a draft beforehand whenever we have some extra to mention.